src/index.ts

```diff
@@ -2249,6 +2256,191 @@ async function claimAllUnclaimedAsSenderPerChain({
 	return txHashes
 }
 
+// Returns args to be passed to makeDepositWithAuthorization function
+// and a EIP-712 message to be signed
+async function makeGaslessDepositPayload({
+	address,
+	contractVersion,
+	linkDetails,
+	password,
+}: interfaces.IPrepareGaslessDepositParams): Promise<interfaces.IMakeGaslessDepositPayloadResponse> {
+	if (!PeanutsWithEIP3009.includes(contractVersion)) {
```

Comment:
why peanuts plural? i think standard name we're using is Peanut

not terribly important since internal nitpick

Reply:
Or is this because PeanutsWithEIP3009 is a dict of various peanut vault versions?

Reply:
> Or is this because PeanutsWithEIP3009 is a dict of various peanut vault versions?

Yeah, it's intended to be a list